system-addon/content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -34,7 +34,7 @@ export class ContextMenu extends React.PureComponent {
       <ul role="menu" className="context-menu-list">
         {this.props.options.map((option, i) => (option.type === "separator" ?
           (<li key={i} className="separator" />) :
-          (<ContextMenuItem key={i} option={option} hideContext={this.hideContext} />)
+          (option.type !== "empty" && <ContextMenuItem key={i} option={option} hideContext={this.hideContext} />)
```

Comment:
I'm assuming some of this is going to change anyways with weighted highlights, since we'll have to have a way to assign context menu options on a per card basis and we won't need this empty stuff anymore